.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,19 @@
+name: pre-commit
```

Comment:
should this be called "check formatting" or something like that?

Reply:
+1